.gitignore

```diff
@@ -1,5 +1,3 @@
-.*
-
```

Comment:
Due tot he file being a ".github" you need to make changes to your git ignore file for the changes to be picked up. This might not be the case in all repositories. Also make extra care that nothing that shouldn't be added gets included as an accidental side effect.

Reply:
Instead of deleting the .* , please just add !.github instead!